.drone.star

```diff
@@ -754,6 +754,8 @@ def ui(config):
       'LOCAL_MAILHOG_HOST': 'email',
     },
     'commands': [
+      'curl -k -s -u admin:admin http://server:8080/ocs/v2.php/apps/testing/api/v1/occ -d "command=app:disable theme-enterprise"',
+      'curl -k -s -u admin:admin http://server:8080/ocs/v2.php/apps/testing/api/v1/occ -d "command=config:system:set --type boolean --value false grace_period.demo_key.show_popup"',
```

Comment:
Note: these commands are not needed any more. The commits can be dropped.
grace period popup is disabled in the core acceptance test script nowadays.
And theme-enterprise has been fixed up to work properly with the new 10.5.0 license mechanism.

See example PR #193 

(but it does not fix the fail of arm64v8)